lib/authenticator.js

```diff
@@ -245,51 +252,54 @@ Authenticator.prototype.sessionManager = function(mgr) {
  *
  * @api public
  */
-Authenticator.prototype.serializeUser = function(fn, req, done) {
+
+// eslint-disable-next-line consistent-return
+Authenticator.prototype.serializeUser = function serializeUser(fn, req, done) {
   if (typeof fn === 'function') {
     return this._serializers.push(fn);
   }
-  
+
   // private implementation that traverses the chain of serializers, attempting
   // to serialize a user
-  var user = fn;
+  const user = fn;
 
   // For backwards compatibility
   if (typeof req === 'function') {
     done = req;
     req = undefined;
   }
-  
-  var stack = this._serializers;
+
+  const stack = this._serializers;
+  // eslint-disable-next-line consistent-return
```

Comment:
If you `return` lines 274, 295, 297 you can get rid of the lint disable for `consistent-return` here and the one above it.

Reply:
If you look at all the places where this has been disabled there's almost always something that can be returned. This is at the point where the function does an implicit return. Where necessary you can add:

```javascript
  return undefined; // for lint
```

I usually add the `// for lint` comment to show why I'm adding a redundant statement. I think this is better than disabling this lint error.